test/Eto.Test.Wpf/Eto.Test.Wpf.csproj

```diff
@@ -18,7 +18,6 @@
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <Prefer32Bit>false</Prefer32Bit>
     <UseWPF Condition="$(HaveWindowsDesktopSdk) == 'True'">true</UseWPF>
-    <UseWindowsForms Condition="$(HaveWindowsDesktopSdk) == 'True'">true</UseWindowsForms>
```

Comment:
This appears to cause a build error, as Eto.Test.Wpf uses both WinForms and WPF for its unit tests.